server/server.js

```diff
@@ -31,7 +31,7 @@ server.on("listening", onListening);
  * Event listener for HTTP server "error" event.
  */
 
-function onError(error) {
+onError = (error) => {
```

Comment:
shouldn't have parenthesis when doing single param.
```suggestion
onError = error => {
```